src/components/ContentHighlights/DeleteHighlightSet.jsx

```diff
@@ -128,15 +154,31 @@ const DeleteHighlightSet = ({ enterpriseId, enterpriseSlug }) => {
           onClose={() => setDeletionError(null)}
           variant="danger"
           dismissible
+          closeLabel={intl.formatMessage({
+            id: 'highlights.modal.delete.highlight.error.dismiss.button.text',
+            defaultMessage: 'Dismiss',
+            description: 'Dismiss button label for error alert shown when deleting a highlight fails.',
+          })}
           icon={Info}
         >
           <p>
-            An error occurred while deleting this highlight collection. Please try again.
+            <FormattedMessage
+              id="highlights.modal.delete.highlight.error.message"
+              defaultMessage="An error occurred while deleting this highlight collection. Please try again."
+              description="Error message shown when deleting a highlight fails."
+            />
           </p>
         </Alert>
         <p>
-          Deleting this highlight will remove it from your
-          learners&apos; &quot;Find a Course&quot;. This action is permanent and cannot be undone.
+          <FormattedMessage
+            id="highlights.modal.delete.highlight.confirmation.message"
+            defaultMessage="Deleting this highlight will remove it from your learners{apostrophe} {doubleQoute}Find a Course{doubleQoute}. This action is permanent and cannot be undone."
+            description="Confirmation message shown when deleting a highlight."
```

Comment:
can't we directly mention apostrophe and doubleQoute? what kind of problem we are getting?

Reply:
this text contains both the single and double quotes. Either we should escaping by `\` or use `values`